content/blog/2022/03/camel316-whatsnew/index.md

```diff
@@ -0,0 +1,108 @@
+---
+title: "Apache Camel 3.16 What's New"
+date: 2022-03-28
+authors: [davsclaus]
+categories: ["Releases"]
+preview: Details of what we have done in the Camel 3.16 release.
+---
+
+Apache Camel 3.16 has just been [released](/blog/2022/03/RELEASE-3.16.0/).
+
+This release introduces a set of new features and noticeable improvements that we will cover in this blog post.
+
+### Resume from Offset
+
+TODO:
+https://camel.apache.org/blog/2022/03/resume-api-v2/
```

Comment:
This release brings a new API to simplify consuming data at scale: the resume API V2. Please check the [blog post](https://camel.apache.org/blog/2022/03/resume-api-v2/) we wrote to introduce it to our community.